templates/core/terraform/airlock/eventgrid_topics.tf

```diff
@@ -31,7 +31,7 @@ resource "azurerm_private_endpoint" "egpe" {
   name                = "pe-eg-${var.tre_id}"
   location            = var.location
   resource_group_name = var.resource_group_name
-  subnet_id           = var.shared_subnet_id
+  subnet_id           = var.airlock_storage_subnet_id
```

Comment:
Does the api function with the new subnet?

Reply:
the function is with a new subnet . (notice I've craeted 2 subnets in this pr)